### PR TITLE
add(workshop): 01-bootrapping-solution

### PR DESCRIPTION
**Pull request by KOCNS1**
Missing slides, here's just the first exercise.

### CODE REVIEW

File: 01-bootstrapping-solution/src/app/app.module.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+/* Your styles goes here. */
```
Comment (Nicoss54) [inline 01-bootstrapping-solution/src/app/app.module.scss]: suggestion(if-minor): remove this file as it is not used

File: package-lock.json [hunk 2]
```diff
@@ -12167,41 +12167,6 @@
         "node": "^14.15.0 || ^16.10.0 || >=18.0.0"
       }
     },
```
Comment (Nicoss54) [inline package-lock.json:12169]: issue(blocking): why package-lock.json change without the package.json change. Seems strange ? Did you rebase the branch before making your Pull request

**Review by Nicoss54 — COMMENTED** [review]
Test failed ! Could you try to fix it please

**Review by Nicoss54 — COMMENTED** [review]
Test failed ! Could you try to fix it please

File: package-lock.json (hunk 2 above)
Comment (KOCNS1) [reply package-lock.json:12169]: Noticed it too, and I did rebase. Can't tell why it triggers a change

**Comment by Nicoss54** [comment]
@KOCNS1 Think to squash all your commit, it will be more pretty to merge juste one commit :)

**Pull request merged**